addons/mail/static/src/components/call_action_list/call_action_list.scss

```diff
@@ -0,0 +1,4 @@
+.o_CallActionList_button:not(.btn-danger) {
+    background-color: var(--CallActionList_button-background-color, #{$o-gray-800});
```

Comment:
Same remark, Not sure to understand, background-color can change but not the font color?

Reply:
Same : https://github.com/odoo/odoo/pull/99755#discussion_r987792256